cmd/resources/gen_resources_test.go

```diff
@@ -0,0 +1,51 @@
+package resources_test
+
+import (
+	"encoding/json"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"ldcli/cmd/resources"
+	"os"
+	"reflect"
+	"testing"
+)
+
+func TestGetTemplateData(t *testing.T) {
+	actual, err := resources.GetTemplateData("test_data/test-openapi.json")
+	assert.NoError(t, err)
+
+	expectedFromFile, err := os.ReadFile("test_data/expected_template_data.json")
+	require.NoError(t, err)
+
+	var expected resources.TemplateData
+	err = json.Unmarshal(expectedFromFile, &expected)
+	require.NoError(t, err)
+
+	t.Run("succeeds with single get resource", func(t *testing.T) {
```

Comment:
not running the comparison directly on each op led to flakey tests - if there's a better way to do this LMK!

Reply:
I'm not getting test failures when running `go test ./cmd/resources/ -run TestGetTemplateData -count=100` after changing the test to
```
t.Run("succeeds with single get resource", func(t *testing.T) {
	assert.Equal(t, expected, actual)
})
```
What failures were you seeing?

Reply:
...I swear I had some. But you're right, updated.